frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.tsx

```diff
@@ -98,45 +97,60 @@ function _BaseBucketPickerPopover({
       <Popover.Target>
         <TriggerButton
           aria-label={triggerLabel}
-          // Compat with E2E tests around MLv1-based components
-          // Prefer using a11y role selectors
           data-testid="dimension-list-item-binning"
           onClick={event => {
             event.stopPropagation();
             setIsOpened(!isOpened);
           }}
+          px="sm"
+          miw="35%"
+          maw="50%"
+          py={0}
+          variant="subtle"
+          color="white"
+          styles={{ label: { display: "flex", gap: "0.5rem" } }}
         >
           <Ellipsified>
             {renderTriggerContent(triggerContentBucketDisplayInfo)}
           </Ellipsified>
           {hasArrowIcon && !hasChevronDown && (
             <TriggerIcon name="chevronright" />
           )}
-          {hasChevronDown && <ChevronDown name="chevrondown" />}
+          {hasChevronDown && (
+            <ChevronDown name="chevrondown" c="transparent !important" />
```

Comment:
Basically, the styles for the Mantine button are either so messed up and inflexible that we need to do this. I _think_ this will be more easily solved with the migration to v7 but we'll have to wait and see